lib/charty/color_methods.rb

```diff
@@ -0,0 +1,85 @@
+module Charty
+  module ColorMethods
+    def RGB(*args)
+      case args.length
+      when 1
+        begin
+          return args[0].to_rgb
+        rescue NoMethodError
+          raise ArgumentError, "the argument must be a color"
+        end
+      when 3
+        return Colors::RGB.new(*args)
+      end
+      raise ArgumentError,
+            "wrong number of arguments (#{args.length} for 1 or 3)"
```

Comment:
Using `"wrong number of arguments (given #{args.length}, expected 1,3)` is better because Ruby uses this format.